crates/next-core/src/next_config.rs

```diff
@@ -1721,6 +1723,29 @@ impl NextConfig {
     pub fn output_file_tracing_excludes(&self) -> Vc<OptionJsonValue> {
         Vc::cell(self.output_file_tracing_excludes.clone())
     }
+
+    #[turbo_tasks::function]
+    pub async fn reqwest_client_config(
+        &self,
+        env: Vc<Box<dyn ProcessEnv>>,
+    ) -> Result<Vc<ReqwestClientConfig>> {
+        let use_system_tls_certs = env
```

Comment:
why support both an experimental option and an env var?

is this a common pattern?

Reply:
i guess you document this on the experimental docs

this may be a per-system issue or a per-project issue and you are providing flexibility.

Reply:
```suggestion
        // Support both an env var and the experimental flag to
        // provide more flexibility to developers on locked down
        // systems.
        let use_system_tls_certs = env
```